plutus-core/plutus-core/examples/PlutusCore/Examples/Builtins.hs

```diff
@@ -99,6 +99,7 @@ data ExtensionFun
     | IdAssumeBool
     | IdAssumeCheckBool
     | IdSomeConstantBool
+    | IdIntegerAsBool
```

Comment:
Did you intend to add these? The PR title indicates that it is doc only.

Reply:
Yes, the title suggests that the PR is all about docs and we needed these examples in docs, hence they are added too.